src/components/DeliverableView/DeliverableStatusBadge.tsx

```diff
@@ -0,0 +1,90 @@
+import { useTheme } from '@mui/material';
+import { Badge } from '@terraware/web-components';
+import { BadgeProps } from '@terraware/web-components/components/Badge';
+import { useMemo } from 'react';
+import strings from 'src/strings';
+import { DeliverableStatusType } from 'src/types/Deliverables';
+
+type DeliverableStatusBadgeProps = {
+  status: DeliverableStatusType;
+};
+
+const DeliverableStatusBadge = (props: DeliverableStatusBadgeProps): JSX.Element => {
+  const { status } = props;
+  const theme = useTheme();
+
+  const getBadgePropsForStatus = useMemo(
+    () => (): BadgeProps => {
```

Comment:
You can remove the extra function declaration, `useMemo` usually just returns a value. `useCallback` can be used if you need to memoize a function

Reply:
Not sure what I was thinking here, thanks! 😅